.goreleaser.yaml

```diff
@@ -10,6 +10,7 @@ builds:
       - -s -w -X go.jetpack.io/devbox/build.Version={{.Version}}
       - -s -w -X go.jetpack.io/devbox/build.Commit={{.Commit}}
       - -s -w -X go.jetpack.io/devbox/build.CommitDate={{.CommitDate}}
+      - -s -w -X go.jetpack.io/devbox/build.TelemetryKey={{ .Env.TELEMETRY_KEY }}
```

Comment:
Does this actually need to be kept secret? It'll still be visible in the binary. Can we just put it in a const instead of a linker flag?

Reply:
I'm gonna leave as is for this PR since it's already working, but we can hard-code it in a follow up clean up.

I'm open either way, last time I asked on Slack, folks leaned towards hiding this, so that's why I implemented this way.